scripts/prepare-release.sh

```diff
@@ -17,9 +17,14 @@ case $OLM_TYPE in
     PACKAGE_NAME=managed-api-service
     OPERATOR_TYPE=rhoam
     ;;
+  "multitenant-managed-api-service")
+    PREVIOUS_VERSION=$(grep $OLM_TYPE bundles/$OLM_TYPE/$OLM_TYPE.package.yaml | awk -F v '{print $3}') || echo "No previous version"
+    PACKAGE_NAME=multitenant-managed-api-service
```

Comment:
do we want to bump this too to managed-api-service?

Reply:
Is there a need for the csv name to be multitenant managed api?
The only difference that I can see between RHOAM and MT RHOAM is a couple of rbacs + additional CRD, is there any harm in having same CSV name?

Reply:
No I don't think there's any harm in having the same CSV name. I changed it to reflect this in the latest push.